addons/mail/static/src/components/composer/composer.js

```diff
@@ -340,13 +339,6 @@ export class Composer extends Component {
         }
     }
 
-    /**
-     * @private
-     */
-    _onInputSubject() {
-        this.composer.update({ subjectContent: this._subjectRef.el.value });
-    }
```

Comment:
There is more to remove, search for `subjectContent` and get rid of all of it. (in `useStore`, in `_update`, `_subjectRef`, and `subjectContent` itself and its other usages).